portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +8,14 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.masthead {
+  height: 100vh;
+  text-align: center;
+  color: #adb5bd;
+  background-image: url("/images/header-background.jpg");
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-position: 30% 0%;
+  background-size: cover;
```

Comment:
Please put declarations in alphabetical order
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order